server/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -187,10 +189,12 @@ private synchronized void closeWithTragicEvent(final Exception ex) {
         final long bytesBufferedAfterAdd;
```

Comment:
I think we should also revert the signature change of `add`, allowing us to revert `Translog.add` to its previous slightly simpler form?